conf.yaml

```diff
@@ -1417,10 +1418,22 @@ contents:
                 repo:   enterprise-search-pubs
                 path:   workplace-search-docs
           -
-
     -
-        title:      SIEM: Security Operations with the Elastic Stack
+        title:      Elastic Security
         sections:
+          -
+            title:      Elastic Endpoint Security
+            prefix:     en/endpoint/guide
+            current:    7.6
+            branches:   [ master, 7.x, 7.6 ]
+            index:      docs/en/endpoint/index.asciidoc
+            chunk:      1
+            tags:       Endpoint/Guide
+            subject:    Endpoint
+            sources:
+              -
+                repo:   stack-docs
+                path:   docs/en/endpoint
```

Comment:
Don't know if you need the `sources` values:

```
-
                repo:   docs
                path:   shared/versions/stack/{version}.asciidoc
              -
                repo:   docs
                path:   shared/attributes.asciidoc
```

Reply:
Not at the moment, I don't think. We aren't sharing any attributes or version yet.